gleam/Dockerfile

```
@@ -0,0 +1,18 @@
+{{#language.image}}
+  FROM gleamlang/gleam
```

is there a maintained version of a docker image ?

---

According to https://github.com/gleam-lang/gleam/pkgs/container/gleam it should be 
```suggestion
  FROM ghcr.io/gleam-lang/gleam:v1.6.3-elixir
```

---

or maybe 
```dockerfile
FROM ghcr.io/gleam-lang/gleam:v1.6.3-erlang
```

---

I wish somebody using gleam would give us hints